app/javascript/shared/components/MarkdownEditor.css

```diff
@@ -6,6 +6,15 @@
   min-height: 10.125rem;
 }
 
-.markdown-editor__fullscreen-preview-wrapper>.markdown-editor__fullscreen-preview-editor {
+.markdown-editor__fullscreen-preview-wrapper
+  > .markdown-editor__fullscreen-preview-editor {
   @apply max-w-3xl mx-auto;
 }
+
+.editor {
+  max-height: 12rem;
+}
```

Comment:
There's a small gap between the focus border of the text area and the wrapper div in the editor:

![Screenshot 2022-02-15 130232](https://user-images.githubusercontent.com/98546/154013903-35a99de9-0b20-40b1-9a36-1f30e2733c3f.png)

This should be fixable by adding a vertical-align here:

```suggestion
.editor {
  max-height: 12rem;
  vertical-align: top;
}
```